src/wr_logic_longrange/nodes/obstacle_avoidance.py

```diff
@@ -68,9 +66,30 @@ def initialize() -> None:
     global drive_pub
     global smoothing_constant
     global speed_factor
+    global heading_pub
+    global frameCount
+    global heading_msg
+    global actual_heading_msg
+    global actual_heading_pub
+    global delta_heading_pub
+    global delta_heading_msg
+    global marker
+    global marker_pub
+    global marker_circle
+    global marker_circle_pub
+    global marker_flag
+    global marker_flag_pub
+    global laser_adjuster_pub
+    global wRover
+    global wRover_pub
+    global rviz_sim_delta_heading_pub
+    global rviz_sim_result_pub
+    global rviz_sim_cur_heading_pub
```

Comment:
I believe all of these variables don't exist in this file. Double check and delete the ones that we don't need.